main.py

```diff
@@ -15,6 +15,9 @@
 CACTI = [pygame.image.load(os.path.join('resources', 'cactus_1.png')),
          pygame.image.load(os.path.join('resources', 'cactus_2.png')),
          pygame.image.load(os.path.join('resources', 'cactus_3.png'))]
+PLATFORM = [pygame.transform.flip(pygame.image.load(os.path.join('resources', 'right_platform.png')),True, False),
```

Comment:
Since we are just flipping the right one, we can delete the left one from the folders